examples/pipelines/stabilize_images.pipe.in

```diff
@@ -22,252 +22,304 @@ process input
 # considered successful (value must be between 0.0 and 1.0).
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:percent_match_req   = 0.45
+      percent_match_req   = 0.45
 
 
 # Algorithm to use for 'feature_matcher'.
 # Must be one of the following options:
 #   - homography_guided
-#   - ocv
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:feature_matcher:type = homography_guided
+#   - ocv_brute_force
+#   - ocv_flann_based
+#   - fundamental_matrix_guided
+#   - viscl
+#   - vxl_constrained
+      feature_matcher:type = homography_guided
 
+      block feature_matcher:homography_guided
 # Algorithm to use for 'feature_matcher'.
 # Must be one of the following options:
 #   - homography_guided
-#   - ocv
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:feature_matcher:homography_guided:feature_matcher:type = ocv
-
-# The OpenCV cv::Algorithm type to use for 'matcher'.
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:feature_matcher:homography_guided:feature_matcher:ocv:matcher:type = DescriptorMatcher.FlannBasedMatcher
+#   - ocv_brute_force
+#   - ocv_flann_based
+#   - fundamental_matrix_guided
+#   - viscl
+#   - vxl_constrained
+        feature_matcher1:type = ocv_flann_based
 
 # Algorithm to use for 'homography_estimator'.
 # Must be one of the following options:
 #   - ocv
 #   - vxl
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:feature_matcher:homography_guided:homography_estimator:type = vxl
+        homography_estimator:type = vxl
 
 # The acceptable error distance (in pixels) between warped and measured points
 # to be considered an inlier match.
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:feature_matcher:homography_guided:inlier_scale = 10
+        inlier_scale = 10
 
 # The minimum required inlier point count. If there are less than this many
 # inliers, no matches will be output.
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:feature_matcher:homography_guided:min_required_inlier_count = 0
+        min_required_inlier_count = 0
 
 # The minimum required percentage of inlier points. If the percentage of points
 # considered inliers is less than this amount, no matches will be output.
-  feature_tracker:default:loop_closer:multi_method:method1:bad_frames_only:feature_matcher:homography_guided:min_required_inlier_percent = 0
-
+        min_required_inlier_percent = 0
+      endblock
+    endblock
 
 # Algorithm to use for 'method2'.
 # Must be one of the following options:
 #   - bad_frames_only
+#   - exhaustive
+#   - keyframe
 #   - multi_method
 #   - vxl_homography_guided
-  feature_tracker:default:loop_closer:multi_method:method2:type                                = vxl_homography_guided
+    loop_closer:multi_method:method2:type                                = vxl_homography_guided
```

Comment:
Homography guided loop closure is the old method for loop closure but not nearly as good as "keyframe" loop closure.  I would use either "keyframe" (good loop closure) or "bad_frames_only" (speed, no long-term loop closure), but probably drop vxl_homography_guided.  You could also use "exhaustive" for absolutely best loop closure if you are willing to wait a very very long time for it, but you probably don't want that usually.